certified-connectors/BoldSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1536 @@
+            }
+          }
+        },
+        "summary": "Sender Identity List",
+        "description": "Retrieve a list of available sender identities (Used to send and manage on behalf of another person).",
+        "operationId": "SenderIdentityList",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "PageSize",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "default": 10,
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "Page",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "default": 1,
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "Search",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "advanced"
+          }
+        ]
+      }
+    },
+    "/WebHooks/AddWebHooksAPIForPowerAutomate": {
+      "x-ms-notification-content": {
+        "schema": {
+          "$ref": "#/definitions/WebHookEventOject"
+        }
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Represents webhook subscribed",
+            "schema": {
+              "$ref": "#/definitions/AddWebHooksResponse"
+            },
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "Location URL that can be used to unsubscribe the webhook"
+              }
+            }
+          }
+        },
+        "summary": "Webhook Events",
+        "description": "This triggers documents webhook event.",
+        "operationId": "WebHooks",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "events",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "items": {
+              "type": "string"
+            },
+            "description": "Select webhook event to which flow should be triggered.",
+            "x-ms-summary": "Select Webhook Events",
+            "x-ms-visibility": "important",
+            "enum": [
+              "Sent",
+              "Signed",
+              "Completed",
+              "Declined",
+              "Revoked",
+              "Reassigned",
+              "Expired",
+              "Viewed",
+              "AuthenticationFailed",
+              "DeliveryFailed",
+              "SendFailed",
+              "TemplateSendFailed",
+              "DraftCreated"
+            ]
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "",
```

Comment:
Please use proper title instead of empty value.

Reply:
Added proper title.